ci/Jenkinsfiles/build.groovy

```diff
@@ -885,6 +886,16 @@ pipeline {
       script {
         if (!isPullRequest() && env.DRY_RUN != "true") {
           currentBuild.description = "Build ${VERSION}"
+          if(!hudson.model.Result.FAILURE.equals(currentBuild.getPreviousBuild()?.getResult())) {
+            slackSend(channel: "${SLACK_CHANNEL}", color: "good", message: "Successfully built ${BRANCH_NAME}/${BUILD_NUMBER}: ${BUILD_URL}")
```

Comment:
WDYT of "Successfully built nuxeo/nuxeo ${BRANCH_NAME} #${BUILD_NUMBER}"?

Reply:
went for https://github.com/nuxeo/nuxeo/pull/4290#discussion_r483241260